### PR TITLE
added extended wv_filters

### PR DESCRIPTION
**Pull request by avinash2692**
Added support for extended wv_filters

### CODE REVIEW

File: src/wv_filters.cpp [hunk 1]
```diff
@@ -168,6 +168,837 @@ arma::field<arma::vec> d4_filter() {
...
+//' mb24_filter()
+// [[Rcpp::export]]
+arma::field<arma::vec> mb24_filter() {
+  
+  arma::vec L(1);
+  L(0) = 24.0;
+  
+  arma::vec g(24) ;
+  
+  g(23) = -2.132706e-05;
```
Comment (coatless) [inline src/wv_filters.cpp]: Any reason for the reverse in g(*) order?

Comment (avinash2692) [inline src/wv_filters.cpp]: Original R script had reversed list as input , was just easier to write it backwards.

Avinash

On Feb 23, 2016, at 10:14 AM, James J Balamuta <notifications@github.com<mailto:notifications@github.com>> wrote:


In src/wv_filters.cpp<https://github.com/SMAC-Group/gmwm/pull/141#discussion_r53803305>:

> +//' }
> +//' @details
> +//' This template can be used to increase the amount of filters available for selection.
> +//' @author JJB
> +//' @keywords internal
> +//' @examples
> +//' mb24_filter()
> +// [[Rcpp::export]]
> +arma::field<arma::vec> mb24_filter() {
> +
> +  arma::vec L(1);
> +  L(0) = 24.0;
> +
> +  arma::vec g(24) ;
> +
> +  g(23) = -2.132706e-05;


Any reason for the reverse in g(*) order?

-

**Comment by coatless** [comment]
* checking for missing documentation entries ... WARNING
Undocumented code objects:
  â€˜bl14_filterâ€™ â€˜bl20_filterâ€™ â€˜d16_filterâ€™ â€˜d6_filterâ€™ â€˜d8_filterâ€™
  â€˜fk14_filterâ€™ â€˜fk22_filterâ€™ â€˜fk4_filterâ€™ â€˜fk6_filterâ€™ â€˜fk8_filterâ€™
  â€˜la16_filterâ€™ â€˜la20_filterâ€™ â€˜la8_filterâ€™ â€˜mb16_filterâ€™ â€˜mb24_filterâ€™
  â€˜mb4_filterâ€™ â€˜mb8_filterâ€™ â€˜w4_filterâ€™
All user-level objects in a package should have documentation entries.
See chapter â€˜Writing R documentation filesâ€™ in the â€˜Writing R
Extensionsâ€™ manual.

**Comment by coatless** [comment]
Addresses #141

**Pull request merged**